components/brave_wallet_ui/page/screens/composer_ui/select_token_modal/select_token_modal.tsx

```diff
@@ -444,159 +538,206 @@ export const SelectTokenModal = React.forwardRef<HTMLDivElement, Props>(
-          <Column
-            fullWidth={true}
-            key={account.name}
-          >
-            <AccountSection
-              width='100%'
-              padding='0px 8px 12px 8px'
-              marginBottom={11}
-              justifyContent='space-between'
-            >
-              <Row width='unset'>
-                <AccountNameText textSize='14px'>
-                  {account.name}
-                </AccountNameText>
-                <AccountAddressText textSize='14px'>
-                  {reduceAddress(account.address)}
-                </AccountAddressText>
-              </Row>
-              {selectedSendOption === SendPageTabHashes.token && (
-                <Row width='unset'>
-                  {isLoadingSpotPrices ? (
-                    <LoadingSkeleton
-                      width={80}
-                      height={14}
-                    />
-                  ) : (
-                    <BalanceText textSize='14px'>
-                      {getAccountFiatValue(account)}
-                    </BalanceText>
-                  )}
-                </Row>
-              )}
-            </AccountSection>
-            <Column
-              fullWidth={true}
-              padding='0px 0px 8px 0px'
-            >
-              {getTokensBySearchValue(account).map((token) => (
-                <TokenListItem
-                  token={token}
-                  onClick={() => handleSelectAsset(token, account)}
-                  key={getAssetIdKey(token)}
-                  balance={getBalance(
-                    account.accountId,
-                    token,
-                    tokenBalancesRegistry
-                  )}
-                  isLoadingSpotPrice={isLoadingSpotPrices}
-                  disabledText={
-                    getIsTokenSelected(token, selectedFromToken)
-                      ? 'braveWalletFromToken'
-                      : getIsTokenSelected(token, selectedToToken)
-                      ? 'braveWalletToToken'
-                      : undefined
-                  }
-                  spotPrice={
-                    spotPriceRegistry
-                      ? getTokenPriceAmountFromRegistry(
-                          spotPriceRegistry,
-                          token
-                        ).format()
-                      : ''
-                  }
-                />
-              ))}
-            </Column>
-          </Column>
-        ) : null
+      return (
+        <VirtualizedTokenList
+          onSelectToken={handleSelectAsset}
+          tokenList={tokensBySearchValue}
+          selectedFromToken={selectedFromToken}
+          selectedToToken={selectedToToken}
+          selectingFromOrTo={selectingFromOrTo}
+          spotPriceRegistry={spotPriceRegistry}
+          isLoadingSpotPrices={isLoadingSpotPrices}
+          getAllAccountsWithBalance={(token: BraveWallet.BlockchainToken) =>
+            getAllAccountsWithBalance(token, accounts, tokenBalancesRegistry)
+          }
```

Comment:
`useCallback` to prevent rerenders (sorry if this was the case before)

Reply:
Changed back to a `useCallback` 👍